.gitignore

```diff
@@ -213,3 +213,5 @@ pip-log.txt
 
 #Mr Developer
 .mr.developer.cfg
+
+.idea
```

Comment:
I agree, if you can merge this for now and then I'll un-ignore once I finish this round of work?